braph2genesis/graphs/_MultilayerBU.gen.m

```diff
@@ -90,7 +90,7 @@ In a multilayer binary undirected (BU) graph, layers could have different number
 value = Graph.NONNEGATIVE * ones(layernumber);
 
 %%% ¡prop!
-A (result, cell) is the cell containing the multilayer binary adjacency matrices of the multilayer binary undirected graph.
+A (result, cell) is the cell containing the multilayer binary supra-adjacency matrices of the multilayer binary undirected graph.
```

Comment:
A (result, cell) is the cell containing the binary supra-adjacency **matrix** of the multilayer binary undirected graph.